framework:
  move_group: arm

default_controller: arm_joint_controller
servo_controller: joint_group_position_controller

link: ["link2", "link3", "link4", "link5", "link6", "pick_link"]

max_iter: 10000
safety_distance: 0.30

# 目标规划器参数
#*_zeta 增益，一个比例
target_zeta: 1

# *_alfa 对由末端位置速度转换得到的关节得到数值乘以的比例
# *_att 从那个  python工程    引入的参数，构型空间？大概类似坡度
target_alfa: 5                       #10
dist_att: 10
target_alfa_rot: 0.5
dist_att_config: 0.5

# 允许误差     
# 允许位置误差  位置总误差finisPosition
target_err: 0.02

# *_ori 指的是姿态  姿态总误差 finishOrien
target_err_ori: 5.5                        #0.017444444

# 轨迹跟随器参数
trajectory_zeta: 1.0  #增益
tra_alfa: 0.6                 #对由末端位置速度转换得到的关节得到数值乘以的比例  1.2
tra_dist_att: 1.2

tra_alfa_rot: 0.5          #对由末端姿态速度转换得到的关节得到数值乘以的比例
tra_dist_att_config: 0.5

global_trajectory_att_outer: 0.15         # 外圈
global_trajectory_att_inner: 0.08       # 内圈
# global_trajectory_att_inner: 0.02

# 避障规划器参数
# 障碍物对每个连杆的增益？
obs_eta: [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]

# obs_rep: [0.01, 0.01, 0.01, 0.01, 0.01, 0.01]
# 对算出的速度取一个比例？
obs_alfa: [0.01, 0.01, 0.01, 0.01, 0.01, 0.01]